### DISCUSSION TITLE
Is `pnpm-workspace.yaml` always a required file?

### DISCUSSION THREAD

**Discussion by hyeongrok7874** [pnpm/pnpm#6381] [Q&A] [answered]
I am currently trying to use pnpm as a package manager to create a `single application` rather than a monorepo.

However, there are many projects that have built a monorepo for the majority of famous open source codes, so it is difficult to find examples that are not monorepo.

My project worked fine even without the `pnpm-workspace.yaml`.

So I want to ask if `pnpm-workspace.yaml` is always required?

Thank you for your interest in the question 🙇‍♂️

**Comment by lvqq** [comment, marked as answer]
It's not required for single application

**Reply by hyeongrok7874** [reply]
Thank you for your reply 🙇‍♂️
